### PR TITLE
Update logging shell

### PR DESCRIPTION
**Pull request by CodiePP**
* simplification in the calling of the bracket funcitons: no more access to the logging configuration needed.
* only shows time between epoch validations in EKG

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -36,16 +37,20 @@ extra-deps:
...
   - git: https://github.com/input-output-hk/iohk-monitoring-framework
-    commit: 00413dcad916a41cf392437199cfba7a2e2c76d7
+    commit: f1c4ceef7d7ea6fb4425484c2b19b84048a3549d
     subdirs:
       - contra-tracer
       - iohk-monitoring
 
+  - ekg-prometheus-adapter-0.1.0.4
+  - prometheus-2.1.1
+  - containers-0.5.11.0
```
Comment (ruhatch) [inline stack.yaml:52]: What's this for? Just curious, not a big issue!

**Review by ruhatch — APPROVED** [review]
LGTM

File: stack.yaml (hunk 1 above)
Comment (CodiePP) [reply stack.yaml:52]: that's a new output which was required by devops. they use prometheus for monitoring/alerting.

**Pull request merged**